app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,58 @@
     <string name="help_title">Help</string>
     <string name="help_navigation_title">Navigation</string>
     <string name="help_navigation_text">
-        From the home screen, swipe down to display the favorites panel and swipe up to hide it.
+        Swipe down from the home screen to show favorites, and upwards to hide them.
         \n
-        \nIf you swipe up when the favorites panel is hidden, it will display the complete list of applications.
-        Swipe down from the top of this list to come back to the home screen.
+        \nSwipe up to show all apps if the favorites panel is hidden.
+

```

Comment:
```suggestion
```
Just a detail but an extra line was added in the middle of the paragraph which looks strange, maybe something related to GitHub if you edit the files from the web interface...
It also happened to other places, I will mark them like this one with "Extra line should be removed" comments.